spec/factories/orphan_filters.rb

```diff
@@ -7,7 +7,7 @@ class OrphanFilter < Hash
 # FactoryGirl.build(:orphan_filter, name_option: "contains", orphan: FactoryGirl.create(:orphan, name: "AA")) to overwrite attributes
 FactoryGirl.define do
   factory :orphan_filter do
-    orphan {build_stubbed :orphan}
+    orphan {build_stubbed :orphan_full}
```

Comment:
`:orphan_full` has fields that are present in the orphan filters, `:orphan` doesn't have them all